GNUmakefile

```diff
@@ -227,17 +236,13 @@ test: ## Run the Nomad test suite and/or the Nomad UI test suite
 test-nomad: dev ## Run Nomad test suites
 	@echo "==> Running Nomad test suites:"
 	@NOMAD_TEST_RKT=1 \
-		go test \
-			-v \
+		go test $(if $(VERBOSE),-v) \
 			-cover \
 			-timeout=900s \
-			-tags="nomad_test $(if $(HAS_LXC),lxc)" ./... >test.log ; echo $$? > exit-code
-	@echo "Exit code: $$(cat exit-code)" >> test.log
-	@grep -A1 -- '--- FAIL:' test.log || true
-	@grep '^FAIL' test.log || true
-	@grep -A10 'panic' test.log || true
-	@test "$$TRAVIS" == "true" && cat test.log || true
-	@if [ "$$(cat exit-code)" == "0" ] ; then echo "PASS" ; exit 0 ; else exit 1 ; fi
+			-tags="nomad_test $(if $(HAS_LXC),lxc)" ./... $(if $(VERBOSE), >test.log ; echo $$? > exit-code)
```

Comment:
I wonder if it would be cleaner for all of this to be done in the script?

Reply:
It will be cleaner but the trade off is that its one step away if you want to see what the args for go test were. Right now, only the portion that checks the log file for errors/panic is in the script, which is less interesting than what `go test` ran with.